README.md

```diff
@@ -19,22 +19,24 @@
 | [**Docs**](https://instadeepai.github.io/jumanji)
 ---
 
-<p float="left" align="center">
-  <img src="docs/env_anim/connector.gif" alt="Connector" width="30%" />
-  <img src="docs/env_anim/snake.gif" alt="Snake" width="30%" />
-  <img src="docs/env_anim/cleaner.gif" alt="Cleaner" width="30%" />
-  <img src="docs/env_anim/job_shop.gif" alt="JobShop" width="30%" />
-  <img src="docs/env_anim/bin_pack.gif" alt="BinPack" width="30%" />
-  <img src="docs/env_anim/cvrp.gif" alt="CVRP" width="30%" />
-  <img src="docs/env_anim/multi_cvrp.gif" alt="MultiCVRP" width="30%" />
-  <img src="docs/env_anim/rubiks_cube.gif" alt="RubiksCube" width="30%" />
-  <img src="docs/env_anim/graph_coloring.gif" alt="GraphColoring" width="30%" />
-  <img src="docs/env_anim/game_2048.gif" alt="Game2048" width="30%" />
-  <img src="docs/env_anim/minesweeper.gif" alt="Minesweeper" width="30%" />
-  <img src="docs/env_anim/tetris.gif" alt="Tetris" width="30%" />
-  <img src="docs/env_anim/mmst.gif" alt="MMST" width="30%" />
-  <img src="docs/env_anim/sudoku.gif" alt="Sudoku" width="30%" />
-</p>
+<div align="center">
+  <img src="docs/env_anim/connector.gif" alt="Connector" width="30%" style="margin: 5px;">
+  <img src="docs/env_anim/snake.gif" alt="Snake" width="30%" style="margin: 5px;">
+  <img src="docs/env_anim/cleaner.gif" alt="Cleaner" width="30%" style="margin: 5px;">
+</div>
+
+<div align="center">
+  <img src="docs/env_anim/job_shop.gif" alt="JobShop" width="30%" style="margin: 5px;">
+  <img src="docs/env_anim/bin_pack.gif" alt="BinPack" width="30%" style="margin: 5px;">
+  <img src="docs/env_anim/tetris.gif" alt="Tetris" width="18%" style="margin: 5px 6.5%;">
```

Comment:
Should the dimensions of tetris not be uniform with the rest? the middle row of the grid looks off, maybe this is why?

Reply:
IMO this is an argument to not include Tetris in this iteration

Reply:
I can't tell if Tristan is joking or not :sweat_smile:

Reply:
Fair point Tristan.

Reply:
Updated `Tetris` and `RobotWarehouse` GIFs to be square. I also added all the other environments to have all 18 GIFs.